src/util.cc

```diff
@@ -736,8 +736,9 @@ Local<String> UnionBytes::ToStringChecked(Isolate* isolate) const {
 }
 
 RAIIIsolateWithoutEntering::RAIIIsolateWithoutEntering(const SnapshotData* data)
```

Comment:
This class is unused after we migrated to simdjson for JSON parsing everywhere, probably better to remove in a different PR, but there is likely no need to update it.